app/serializers/serializers/representation_habitats_serializer.rb

```diff
@@ -0,0 +1,32 @@
+class Serializers::RepresentationHabitatsSerializer < Serializers::Base
+
+  def initialize(coastal_stat, habitat_stats)
+    super(coastal_stat, 'coastal_stat')
+    super(habitat_stats, 'habitat_stats')
+  end
+
+  def serialize
+    coastal = @coastal_stat.attributes.slice('total_coast_length', 'multiple_habitat_length')
```

Comment:
This currently breaks for regions pages as we don't have a coastal_stat object related to those